src/lib/modules/blockchain_connector/provider.js

```diff
@@ -16,9 +20,48 @@ class Provider {
 
   startWeb3Provider(callback) {
     const self = this;
-
+    
     if (this.type === 'rpc') {
       self.provider = new this.web3.providers.HttpProvider(self.web3Endpoint);
+    } else if (this.type === 'ledger') {
+        let hardwareWalletConfigs = [];
+
+        self.accountsConfig.forEach((accountConfig) => {
+          if (accountConfig.hardwareWallet) {
+            if (accountConfig.hardwareWallet === 'ledger') {
+              hardwareWalletConfigs.push(accountConfig.hardwareWallet);
+            }
+          }
+        });
+
+        if (hardwareWalletConfigs.length > 0) {
+          if (hardwareWalletConfigs.length > 1) {
+            self.logger.warn('Mulitple ledger wallet configs found, so selecting the first config only.');
```

Comment:
```suggestion
            self.logger.warn(__('Multiple ledger wallet configs found. Selecting the first config only.'));
```
Fixed a type and also added `__` as it handles translations.